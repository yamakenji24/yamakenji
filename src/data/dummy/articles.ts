import { OGPDataType } from '../../services/get-ogp';

export const dummyArticles: Array<OGPDataType> = Array(10)
  .fill(0)
  .map((_, idx) => ({
    url: '/not_found.png',
    title: 'dummy title',
    image: {
      height: '200',
      type: 'type',
      url: '/not_found.png',
      width: '200',
    }
  }))