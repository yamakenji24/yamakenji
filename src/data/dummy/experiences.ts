import { ExperienceType } from 'services/get-experience-api';

export const dummyExperiences: Array<ExperienceType> = Array(10)
  .fill(0)
  .map((_, idx) => ({
    id: idx.toString(),
    during: '',
    title: 'dummy experience',
    body: 'dummy body',
  }));
