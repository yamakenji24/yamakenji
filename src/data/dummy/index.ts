export * from './experiences';
export * from './skills';
