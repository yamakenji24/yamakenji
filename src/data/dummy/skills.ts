import { SkillType } from 'services/get-skill-api';

export const dummySkills: Array<SkillType> = Array(10)
  .fill(0)
  .map((_, idx) => ({
    id: idx.toString(),
    name: 'dummy skill',
    img: '/not_found.png',
  }));
