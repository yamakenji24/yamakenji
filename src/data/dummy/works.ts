import { WorkType } from 'services/get-work-api';

export const dummyWorks: Array<WorkType> = Array(10)
  .fill(0)
  .map((_, idx) => ({
    id: idx.toString(),
    github: '',
    skills: [{
      fieldId: '',
      skill: '',
    }],
    image: {
      height: '200',
      type: 'type',
      url: '/not_found.png',
      width: '200',
    },
    title: 'dummy title',
    url: '/not_found.png',
    description: 'dummy description'
  }))