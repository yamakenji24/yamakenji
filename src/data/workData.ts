export type Works = {
  github: string;
  skills: string[];
  url: string;
  body: string;
  title: string;
};

export const works: Works[] = [
  {
    github: 'https://github.com/yamakenji24/blog.yamakenji',
    skills: ['Remix', 'React', 'Tailwind', 'TypeScript'],
    url: 'https://blog.yamakenji.com/',
    body: 'Remixを用いたブログを開設しました。技術的メモや参加報告、ブログ的な内容はここで紹介していきます。',
    title: 'yamakenji blog',
  },
  {
    github: 'https://github.com/yamakenji24/CardCreator',
    skills: ['Next.js', 'React', 'TypeScript', 'Tailwind'],
    url: 'https://card-creator-hp6vcyu1c.vercel.app/',
    body: 'Next.jsとTailwindを用いたエンジニア用名刺アプリ。Next.js conf2020のチケットを参考に作成',
    title: 'エンジニア名刺アプリ',
  },
  {
    github: 'https://github.com/yamakenji24/yamakenji',
    skills: ['Next.js', 'React', 'TypeScript', 'Chakra UI'],
    url: 'https://yamakenji.com/',
    body: 'このプロフィールサイトです。Next.jsを利用して、作成しています。製作物や記事など、随時更新していきます',
    title: 'yamakenji24 profile',
  },
  {
    github: 'https://github.com/yamakenji24/SpotifyPlayer',
    skills: ['React', 'Redux', 'TypeScript'],
    url: 'https://yamakenji24.github.io/SpotifyPlayer/',
    body: 'Reactとspotify apiを用いた音楽検索アプリ。masterブランチでは、classコンポーネントとredux-sagaを利用して実装。hook-verブランチでは、hooksとtypescriptを利用',
    title: 'Spotify APIを用いた音楽検索アプリ',
  },
];
