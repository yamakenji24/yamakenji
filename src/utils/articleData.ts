export interface ArticleData {
  url: string;
  title: string;
  img: string;
  date: string;
}
export const articleData: Array<ArticleData> = [
  {
    url: 'https://zenn.dev/yamakenji24/articles/2921714d4b69d0',
    title: 'Next.jsとTailwindを利用したエンジニア名刺アプリ',
    img: '/article/cardcreator.jpg',
    date: '2020/12',
  },
  {
    url: 'https://yamakenji24.hatenablog.com/entry/2020/11/13/192543',
    title: 'Chatworkのサマーインターン2020に参加してきた',
    img: '/article/intern.jpg',
    date: '2020/09',
  },
  {
    url: 'https://shell-mag.com/vol-68/',
    title: 'シェルスクリプトマガジン Vol.68',
    img: '/article/shellvol68.jpg',
    date: '2020/08',
  },
  {
    url: 'https://qiita.com/yamakenji/items/38fd1175bdd3338fb810',
    title: 'GraphQLについてまとめてみた',
    img: '/article/GraphQL.jpg',
    date: '2020/06/23',
  },
  {
    url: 'https://qiita.com/yamakenji/items/b94fa2ab104679b29b3a',
    title: 'React+Reduxで音楽検索アプリを作ってみた',
    img: '/article/spotify.jpg',
    date: '2019/12/14',
  },
  {
    url: 'https://shell-mag.com/vol-60/',
    title: 'シェルスクリプトマガジン Vol.60',
    img: '/article/shellvol60.jpg',
    date: '2019/06',
  },
];
