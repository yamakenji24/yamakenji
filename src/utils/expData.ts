export interface ExpData {
  during: string;
  title: string;
  body: string;
}

export const expData: Array<ExpData> = [
  {
    during: 'September 14th, 2020 ~ September 18th, 2020',
    title: 'Monotaro Intern',
    body: 'インターンに参加 ~',
  },
  {
    during: 'August 17th, 2020 ~ September 4th, 2020',
    title: 'ChatWork Intern',
    body: 'インターンに参加 ~',
  },
  {
    during: 'April 2020 - present',
    title: 'Kagawa University',
    body: 'Master: Division of Reliability-based Information Systems Engineering',
  },
  {
    during: 'April 2016 ~ March 2020',
    title: 'Kagawa University',
    body: 'Bachelor: Faculty of Engineering',
  },
];
