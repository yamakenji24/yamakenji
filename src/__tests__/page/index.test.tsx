import React from 'react';
import { render } from '../utils';

import Toppage from 'pages/index';
import { dummyExperiences, dummySkills } from 'data/dummy';

test('/', () => {
  const { container } = render(
    <Toppage experiences={dummyExperiences} skills={dummySkills} ogImageUrl={''} />,
  );
  expect(container).toMatchSnapshot();
});
