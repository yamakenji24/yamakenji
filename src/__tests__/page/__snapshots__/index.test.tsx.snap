// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/ 1`] = `
<div>
  <div
    class="css-0"
  >
    <nav
      class="css-1cahzu1"
    >
      <div
        class="css-cpn6u3"
      >
        <span
          style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2780%27%20height=%2760%27/%3e"
              style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
            />
          </span>
          <img
            data-nimg="intrinsic"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
          <noscript />
        </span>
      </div>
      <div
        class="css-1c2za90"
      >
        <p
          class="chakra-text css-v05hfz"
        >
          <a
            class="chakra-link css-3wpten"
            data-test-id="navigation-Top"
          >
            Top
          </a>
        </p>
        <p
          class="chakra-text css-v05hfz"
        >
          <a
            class="chakra-link css-3wpten"
            data-test-id="navigation-Work"
          >
            Work
          </a>
        </p>
        <p
          class="chakra-text css-v05hfz"
        >
          <a
            class="chakra-link css-3wpten"
            data-test-id="navigation-Article"
          >
            Article
          </a>
        </p>
        <p
          class="chakra-text css-v05hfz"
        >
          <a
            class="chakra-link css-3wpten"
            data-test-id="navigation-Contact"
          >
            Contact
          </a>
        </p>
      </div>
    </nav>
    <div
      class="css-rszk63"
    >
      <div
        class="css-0"
      >
        <div
          class="css-cowfjb"
        >
          <span
            alt="Kenji Yamashita"
            class="chakra-avatar css-1t35cdq"
          >
            <svg
              aria-label=" avatar"
              class="chakra-avatar__svg css-16ite8i"
              role="img"
              viewBox="0 0 128 128"
            >
              <path
                d="M103,102.1388 C93.094,111.92 79.3504,118 64.1638,118 C48.8056,118 34.9294,111.768 25,101.7892 L25,95.2 C25,86.8096 31.981,80 40.6,80 L87.4,80 C96.019,80 103,86.8096 103,95.2 L103,102.1388 Z"
                fill="currentColor"
              />
              <path
                d="M63.9961647,24 C51.2938136,24 41,34.2938136 41,46.9961647 C41,59.7061864 51.2938136,70 63.9961647,70 C76.6985159,70 87,59.7061864 87,46.9961647 C87,34.2938136 76.6985159,24 63.9961647,24"
                fill="currentColor"
              />
            </svg>
          </span>
          <div
            class="css-acsbfs"
          >
            <p
              class="chakra-text css-1qfh309"
            >
              Kenji Yamashita
            </p>
            <div
              class="css-mq43ny"
            >
              <p
                class="chakra-text css-0"
              >
                @yamakenji24
              </p>
              <p
                class="chakra-text css-0"
              >
                某うどん県で大学院に通っている学生です。
              </p>
              <p
                class="chakra-text css-0"
              >
                Master student
              </p>
            </div>
          </div>
        </div>
        <div
          class="css-1hg94rl"
        >
          <div
            class="css-1mu03uk"
          >
            <a
              class="chakra-link css-f4h6uy"
              href="https://github.com/yamakenji24"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2732%27%20height=%2732%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </a>
          </div>
          <div
            class="css-1mu03uk"
          >
            <a
              class="chakra-link css-f4h6uy"
              href="https://yamakenji24.hatenablog.com/"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2732%27%20height=%2732%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </a>
          </div>
          <div
            class="css-1mu03uk"
          >
            <a
              class="chakra-link css-f4h6uy"
              href="https://qiita.com/yamakenji"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2732%27%20height=%2732%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </a>
          </div>
          <div
            class="css-1mu03uk"
          >
            <a
              class="chakra-link css-f4h6uy"
              href="https://zenn.dev/yamakenji24"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2732%27%20height=%2732%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </a>
          </div>
        </div>
      </div>
      <div
        class="css-j7qwjs"
      >
        <h4
          class="chakra-heading css-1sr6c4u"
        >
          Experience
        </h4>
        <div
          class="css-fb8b55"
        >
          <div
            class="css-90hjgt"
          >
            <p
              class="chakra-text css-1qm1lh"
            />
            <p
              class="chakra-text css-r4q4lh"
            >
              dummy experience
            </p>
            <p
              class="chakra-text css-1sr8fdr"
            >
              dummy body
            </p>
          </div>
          <div
            class="css-90hjgt"
          >
            <p
              class="chakra-text css-1qm1lh"
            />
            <p
              class="chakra-text css-r4q4lh"
            >
              dummy experience
            </p>
            <p
              class="chakra-text css-1sr8fdr"
            >
              dummy body
            </p>
          </div>
          <div
            class="css-90hjgt"
          >
            <p
              class="chakra-text css-1qm1lh"
            />
            <p
              class="chakra-text css-r4q4lh"
            >
              dummy experience
            </p>
            <p
              class="chakra-text css-1sr8fdr"
            >
              dummy body
            </p>
          </div>
          <div
            class="css-90hjgt"
          >
            <p
              class="chakra-text css-1qm1lh"
            />
            <p
              class="chakra-text css-r4q4lh"
            >
              dummy experience
            </p>
            <p
              class="chakra-text css-1sr8fdr"
            >
              dummy body
            </p>
          </div>
          <div
            class="css-90hjgt"
          >
            <p
              class="chakra-text css-1qm1lh"
            />
            <p
              class="chakra-text css-r4q4lh"
            >
              dummy experience
            </p>
            <p
              class="chakra-text css-1sr8fdr"
            >
              dummy body
            </p>
          </div>
          <div
            class="css-90hjgt"
          >
            <p
              class="chakra-text css-1qm1lh"
            />
            <p
              class="chakra-text css-r4q4lh"
            >
              dummy experience
            </p>
            <p
              class="chakra-text css-1sr8fdr"
            >
              dummy body
            </p>
          </div>
          <div
            class="css-90hjgt"
          >
            <p
              class="chakra-text css-1qm1lh"
            />
            <p
              class="chakra-text css-r4q4lh"
            >
              dummy experience
            </p>
            <p
              class="chakra-text css-1sr8fdr"
            >
              dummy body
            </p>
          </div>
          <div
            class="css-90hjgt"
          >
            <p
              class="chakra-text css-1qm1lh"
            />
            <p
              class="chakra-text css-r4q4lh"
            >
              dummy experience
            </p>
            <p
              class="chakra-text css-1sr8fdr"
            >
              dummy body
            </p>
          </div>
          <div
            class="css-90hjgt"
          >
            <p
              class="chakra-text css-1qm1lh"
            />
            <p
              class="chakra-text css-r4q4lh"
            >
              dummy experience
            </p>
            <p
              class="chakra-text css-1sr8fdr"
            >
              dummy body
            </p>
          </div>
          <div
            class="css-90hjgt"
          >
            <p
              class="chakra-text css-1qm1lh"
            />
            <p
              class="chakra-text css-r4q4lh"
            >
              dummy experience
            </p>
            <p
              class="chakra-text css-1sr8fdr"
            >
              dummy body
            </p>
          </div>
        </div>
      </div>
      <div
        class="css-j7qwjs"
      >
        <h4
          class="chakra-heading css-1sr6c4u"
        >
          Skills
        </h4>
        <div
          class="css-1dlcyqs"
        >
          <div
            class="css-1sif8c9"
          >
            <div
              class="css-gmuwbf"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2760%27%20height=%2745%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </div>
            <p
              class="chakra-text css-44cqda"
            >
              dummy skill
            </p>
          </div>
          <div
            class="css-1sif8c9"
          >
            <div
              class="css-gmuwbf"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2760%27%20height=%2745%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </div>
            <p
              class="chakra-text css-44cqda"
            >
              dummy skill
            </p>
          </div>
          <div
            class="css-1sif8c9"
          >
            <div
              class="css-gmuwbf"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2760%27%20height=%2745%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </div>
            <p
              class="chakra-text css-44cqda"
            >
              dummy skill
            </p>
          </div>
          <div
            class="css-1sif8c9"
          >
            <div
              class="css-gmuwbf"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2760%27%20height=%2745%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </div>
            <p
              class="chakra-text css-44cqda"
            >
              dummy skill
            </p>
          </div>
          <div
            class="css-1sif8c9"
          >
            <div
              class="css-gmuwbf"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2760%27%20height=%2745%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </div>
            <p
              class="chakra-text css-44cqda"
            >
              dummy skill
            </p>
          </div>
          <div
            class="css-1sif8c9"
          >
            <div
              class="css-gmuwbf"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2760%27%20height=%2745%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </div>
            <p
              class="chakra-text css-44cqda"
            >
              dummy skill
            </p>
          </div>
          <div
            class="css-1sif8c9"
          >
            <div
              class="css-gmuwbf"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2760%27%20height=%2745%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </div>
            <p
              class="chakra-text css-44cqda"
            >
              dummy skill
            </p>
          </div>
          <div
            class="css-1sif8c9"
          >
            <div
              class="css-gmuwbf"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2760%27%20height=%2745%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </div>
            <p
              class="chakra-text css-44cqda"
            >
              dummy skill
            </p>
          </div>
          <div
            class="css-1sif8c9"
          >
            <div
              class="css-gmuwbf"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2760%27%20height=%2745%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </div>
            <p
              class="chakra-text css-44cqda"
            >
              dummy skill
            </p>
          </div>
          <div
            class="css-1sif8c9"
          >
            <div
              class="css-gmuwbf"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2760%27%20height=%2745%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </div>
            <p
              class="chakra-text css-44cqda"
            >
              dummy skill
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
